s2i/copy_mvn_repo.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+VERSION="$1"
+TARGET_DIR="$2"
+
+XML="""
+  <distributionManagement>
+    <snapshotRepository>
+      <id>local</id>
+      <url>file:/deployments/project/.m2/repository</url>
+    </snapshotRepository>
+    <repository>
+      <id>local</id>
+      <url>file:/deployments/project/.m2/repository</url>
+    </repository>
+  </distributionManagement>
+"""
+perl -p -i -e "s|</project>|${XML}</project>|" "$TARGET_DIR/pom.xml"
```

Comment:
Love it ;-) But think sed could be good  enough here, too (so that it works for systems without perl, too).

Reply:
But I'm fine with this of course (actually I would rewrite it into a complete perl script or maybe even groovy script to make it cross plattform)

Reply:
However, the script has a serious flaw: You will add endlessly `<distributionManagement>` sections to `src/main/docker/pom.xml` regardless of whether this section is already included or not. Also modifying committed source files has issues (remember the connectors ?)

Reply:
the pom.xml is generated on every mvn build, so no endless dups of '<distributionManagement>'.   Tried sed.  It fails to handle '\n''s well.  Our build.sh is based on bash already.  Seem we are already tied down to bash based systems.